docs/networking.md

```diff
@@ -191,6 +191,15 @@ For help with Calico or to report any issues:
 
 Calico currently uses etcd as a backend for storing information about workloads and policies.  Calico does not interfere with normal etcd operations and does not require special handling when upgrading etcd.  For more information please visit the [etcd Docs](https://coreos.com/etcd/docs/latest/)
 
+#### Calico troubleshooting
+##### New nodes are taking minutes for syncing ip routes and new pods on them can't reach kubedns
+This is caused by nodes in the Calico etcd nodestore no longer existing. Due to the ephemeral nature of AWS EC2 instances, new nodes are brought up with different hostnames, and nodes that are taken offline remain in the Calico nodestore. This is unlike most datacentre deployments where the hostnames are mostly static in a cluster. Read more about this issue at https://github.com/kubernetes/kops/issues/3224
+This has been solved in kops 1.8.2, when creating a new cluster no action is needed, but if the cluster was created with a prior kops version the following actions should be taken:
+  * Use kops to update the cluster ```kops update cluster <name> --yes``` and wait for calico-kube-controllers deployment and calico-node daemonset to be updated
+  * Delete all calico-node pods in kube-system namespace, so that they will be recreated with the new env CALICO_K8S_NODE_REF and update the current nodes in etcd
```

Comment:
Can we give kubectl command?

Reply:
I did it manually and right now I have no time thinking of a command that would do everything :/, maybe @mikesplain or @caseydavenport  have a script for that?

Reply:
Actually, I would suggest just rolling over the whole cluster... Deleting the pods could create temporary connectivity loss couldn't it? We drain and roll the cluster anytime we need to change core daemonsets like this.

Reply:
not actually, I always do ```kubectl delete pods -n kube-system -l "k8s-app=calico-node" --force --grace-period=0```
deleting calico-node pods does not delete the iptables rules and interfaces that were already created, so no problem on that.

Reply:
@chrislovecnm you mean kubectl command to delete calico-node pods or to delete the invalid nodes in etcd storage, the later is the one I have no time to provide, the first one is in my comment above

Reply:
@felipejfc Fair enough! I wasn't certain. I have a script/container open sourced that we could use: https://github.com/mikesplain/calico-clean/blob/master/calico-clean.sh

Would just need to convert https://github.com/mikesplain/calico-clean/blob/master/CronJob.yaml into a one time job.

Reply:
Do we need to delete the calico-node pods? Won't upgrade the cluster trigger a rolling update due to the env change?

If not couldn't we enable rolling update on the daemonset and then it would?

Deleting all workload pods shouldn't be necessary (nor should rolling the whole cluster).

Reply:
I aggre it should be ok to add a rolling update strategy @caseydavenport, what do you guys think @mikesplain @chrislovecnm @robinpercy

Reply:
Personally, I've never used rolling update strategy on daemonset.  I could see cases when upgrading an entire cluster would trigger a daemonset upgrade which may not be intended on older nodes... We could put it behind a config value or something.  I have mixed feelings... Thoughts @chrislovecnm @robinpercy ?

Reply:
It should only trigger when changes are made to the DaemonSet (not other things in the cluster) so I suspect it's OK and desirable.

We enable it by default in upstream Calico manifests.